### PR TITLE
add deployment memory limits

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees PTAL

### CODE REVIEW

File: openshift/templates/cakephp-mysql.json [hunk 1]
```diff
@@ -314,75 +324,101 @@
...
       "name": "CAKEPHP_SECURITY_CIPHER_SEED",
+      "displayName": "CakePHP security cipher seed",
       "description": "Security cipher seed for session hash",
       "generate": "expression",
       "from": "[0-9]{30}"
     },
     {
       "name": "OPCACHE_REVALIDATE_FREQ",
+      "displayName": "OPcache revalidation frequency",
       "description": "The How often to check script timestamps for updates, in seconds. 0 will result in OPcache checking for updates on every request.",
```
Comment (bparees) [inline openshift/templates/cakephp-mysql.json]: s/The//  (i know, not part of your changes)

File: openshift/templates/cakephp.json [hunk 2]
```diff
@@ -208,69 +213,89 @@
...
       "name": "CAKEPHP_SECURITY_CIPHER_SEED",
+      "displayName": "CakePHP security cipher seed",
       "description": "Security cipher seed for session hash",
       "generate": "expression",
       "from": "[0-9]{30}"
     },
     {
       "name": "OPCACHE_REVALIDATE_FREQ",
+      "displayName": "OPcache revalidation frequency",
       "description": "The How often to check script timestamps for updates, in seconds. 0 will result in OPcache checking for updates on every request.",
```
Comment (bparees) [inline openshift/templates/cakephp.json]: s/The//

**Comment by bparees** [comment]
2 nits that aren't your fault and lgtm.  (I assume things successfully deploy w/ these limits)

**Comment by gabemontero** [comment]
Will push the 2 nits momentarily.  Yep, lots of deploys with these constraints.  If you are interested in the gory detail around my evaluations, I have a (quasi-organized) spread sheet at https://docs.google.com/spreadsheets/d/1XHXPOrSAGd31ueQN_PH0HIZrNmsfZoXoLpFwL4-ZjvY/edit#gid=0

**Comment by gabemontero** [comment]
edits now pushed

**Pull request merged**